docs/blog/2019-02-20-getting-started-with-gatsby-themes/index.md

```diff
@@ -0,0 +1,300 @@
+---
+title: "Getting Started with Gatsby Themes and MDX"
+date: 2019-02-20
+author: Katie Fujihara
+tags: ["themes", "tutorials"]
+---
+
+### What is a Gatsby theme?
+
+<pullquote cite="Jason Lengstorf">Gatsby themes allow you to focus only on the parts of the site and app building process that you need to care about by abstracting the rest away into a package.</pullquote>
+
+## Getting Started
+
+Create a new directory
+
+`mkdir gatsby-themes`
+
+Navigate to the directory
+
+`cd gatsby-themes`
+
+Create a _package.json_ file
+
+`npm init`
```

Comment:
We should probably keep the usage of `yarn` consistent for this post (since it's required anyway for workspaces).

```suggestion
`yarn init`
```